hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/audit/impl/LoggingAuditor.java

```diff
@@ -230,6 +240,26 @@ private class LoggingAuditSpan extends AbstractAuditSpanImpl {
 
     private final HttpReferrerAuditHeader referrer;
 
+    /**
+     * Attach Range of data for GetObject Request.
+     * @param request given get object request
+     */
+    private void attachRangeFromRequest(AmazonWebServiceRequest request) {
+      if (request instanceof GetObjectRequest) {
+        long[] rangeValue = ((GetObjectRequest) request).getRange();
+        if (rangeValue == null || rangeValue.length == 0) {
+          return;
+        }
+        if (rangeValue.length != 2) {
+          WARN_INCORRECT_RANGE.warn("Expected range to contain 0 or 2 elements. Got "
+              + rangeValue.length + " elements. Ignoring");
+          return;
+        }
+        String combinedRangeValue = String.format("bytes=%d-%d", rangeValue[0], rangeValue[1]);
```

Comment:
can you make a two byte prefix ("rg"?) and put in AuditConstants?
* shorter as we don't know the upper limit on referrer strings and would like to keep it down
* in the file for other connectors to use, and to make it easy to look at what to expect

Reply:
@steveloughran, can you elaborate on the `rg` prefix? Is it for the query parameter? If so, we already have that as `r` (`PARAM_RANGE`).

`bytes=` is part of the range header value which - following the SDK upgrade - can just be dumped in place, without needing reconstruction like this.

That being said, it does seem a bit of a waste. Maybe we drop it entirely here (and for all connectors) and just include whatever the value of the range header was minus `bytes=` prefix.

Reply:
sorry, i'd missed that. yes, you are right. just include it without the bytes= stuff. 

i guess we do have to consider what happens if/when s3 supports multiple ranges, but that would be just something like "1-3,4-7", wouldn't it?

Reply:
Yes, I think that makes sense.

The only risk I see is if the object store uses a range specified in another unit from bytes. In V1 SDK, it's hard coded to `bytes` - no issue there. In V2, I believe you specify the header itself so we can warn once if we see something other than `bytes=` prefix and ignore the header if so. We are the ones specifying the header in the first place anyway, right? (FYI @ahmarsuhail @passaro)